packages/app/src/app/pages/common/Modals/MemberPaymentConfirmation/index.tsx

```diff
@@ -12,7 +12,7 @@ export const MemberPaymentConfirmation: React.FC<{ title: string }> = ({
   const {
     activeTeamInfo,
     personalWorkspaceId,
-    pro: { prices },
+    pro: { legacyPrices },
```

Comment:
Can't we use the `prices` value here?

Reply:
Yes we should, overlooked that one!

Reply:
I've updated it in [2010fa8](https://github.com/codesandbox/codesandbox-client/pull/7451/commits/2010fa88cd5c18cfe343e4a4ba38bc27074b5452)! I've also updated the pro page to use the existing `formatCurrency` function instead of a new function.